src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Changelog
 
-## Unreleased
+## 1.6.0-beta.1
 
-* Release together with `OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule`.
-  Fixed an issue where activities were stopped incorrectly before processing completed.
-  Activity processor's `OnEnd` will now happen after `AspNetInstrumentationOptions.Enrich`.
+Released 2023-Oct-11
+
+* Fixed an issue where activities were stopped incorrectly before processing
+  completed. Activity processor's `OnEnd` will now happen after
+  `AspNetInstrumentationOptions.Enrich`.
   ([#1388](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1388))
```

Comment:
I have some doubts about changelog, technically the only change in OpenTelemetry.Instrumentation.AspNet is the upgrade package version
the fix for enriching and bumping OTel is in the httpmodule package

Maybe we can simplify to:

```suggestion
* Update `OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule` to 1.6.0-beta.1.
  ([#1392](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1392))
```

Reply:
I think it's helpful to have that information about the fix in AspNet CHANGELOG as well. Most users only deal with this library, and it might not occur to them to check the CHANGELOG of the Telemetry Module project.